src/bio/bio_xstream.c

```diff
@@ -1360,6 +1360,11 @@ init_blobstore_ctxt(struct bio_xs_context *ctxt, int tgt_id)
 				DP_RC(rc));
 			goto out;
 		}
+		/* Set the NVMe SSD PCI Vendor ID */
+		bio_set_vendor_id(bbs, d_bdev->bb_name);
+		/* Register DAOS metrics to export NVMe SSD health stats */
+		bio_export_health_stats(bbs, d_bdev->bb_name);
+		bio_export_vendor_health_stats(bbs, d_bdev->bb_name);
```

Comment:
Why don't call these three functions in bio_init_health_monitoring()? So that we don't have to export these monitoring details to other BIO C files.